lib/proto/lndinvoices_pb.js

```diff
@@ -653,7 +655,7 @@ proto.invoicesrpc.AddHoldInvoiceRequest.prototype.getExpiry = function() {
 
 /** @param {number} value */
 proto.invoicesrpc.AddHoldInvoiceRequest.prototype.setExpiry = function(value) {
-  jspb.Message.setField(this, 5, value);
+  jspb.Message.setProto3IntField(this, 5, value);
```

Comment:
Can you check your protoc compiler version? I believe it should be 3.4. There shouldn't be these changes to this file. Ensure you have `grpc-tools` 1.6.0 installed in the xud directory and try regenerating protos, these changes should go away.

Reply:
I do have those versions.
For some reason, this still happens.
I will look into it.